.github/workflows/codeql-analysis.yml

```diff
@@ -6,7 +6,6 @@ on:
       - main
       - release-*
   pull_request:
-    # The branches below must be a subset of the branches above
```

Comment:
this isn't true anymore?

Reply:
just deleting the comments, they're not very useful